plugins/modules/azuread_group.py

```diff
@@ -507,71 +532,57 @@ def compare_groups(current, new):
     current_keys = current.keys()
     new_keys = new.keys()
     current_keys_to_remove = [item for item in current_keys if item not in new_keys]
-    new_keys_to_remove = ["owners", "members"]
     # Remove the unknown keys from remote group
     for item in current_keys_to_remove:
         if item in current:
             current.pop(item)
     # Remove the keys that are not returned by Get method from new group
-    for item in new_keys_to_remove:
-        if item in new:
-            new.pop(item)
     if current != new:
         return dict(before=current, after=new)
 
 
+def build_group_from_playbook(params):
```

Comment:
```suggestion
def build_group_from_module(params):
```

on est loin du playbook